resources/views/components/filter/disclosure_button.blade.php

```diff
@@ -0,0 +1,11 @@
+<button
+    class="tw-flex tw-items-center tw-border tw-border-gray-300 tw-bg-white tw-py-2 tw-px-3 tw-text-base tw-font-medium hover:tw-border-gray-800"
+    {{ $attributes }}>
```

Comment:
This looks very nice 👍 

But we can't pass any more classes to it right now. We can do this:

```suggestion
<button
    {{ $attributes->merge(['class' => "tw-flex tw-items-center tw-border tw-border-gray-300 tw-bg-white tw-py-2 tw-px-3 tw-text-base tw-font-medium hover:tw-border-gray-800"]) }} >
```